### DISCUSSION TITLE
Citation of image of iCub Reference Frames

### DISCUSSION THREAD

**Discussion by diogofbsilva** [robotology/community#716] [Q&A] [answered]
Hello everyone!

I am writing my master thesis, and I really like the image available at https://icub-tech-iit.github.io/documentation/icub_kinematics/icub-forward-kinematics/icub-forward-kinematics/.

I would like to use it in my thesis, is there any work where this image was used that I can cite in order to use the image? Maybe a work from @randaz81 as I am seeing that is name is stated in the image.

Thank you so much for your help and attention.

Regards
Diogo Silva

**Comment by alecive** [comment]
Wasn't that picture made by @jeljaik ?

**Comment by pattacini** [comment, marked as answer]
Hi @diogofbsilva 

If no one shows up pointing to a proper scientific resource, you can refer to the page on the website.

**Comment by jeljaik** [comment]
Hi @diogofbsilva yes, I actually was the original author of that image, if you refer to the one below. I actually made it to have it printed and pinned in front of me during my time at IIT :) So I don't have a specific document where I used it. Glad to see it alive somewhere though. Best of lucks with your PhD. ! 
![](https://icub-tech-iit.github.io/documentation/icub_kinematics/icub-forward-kinematics/img/1104px-NewiCubRefFrames1.png)

**Comment by diogofbsilva** [comment]
Thank you so much for all your help!!

I will follow @pattacini advice and refer to the page on the website.
